### PR TITLE
Add basic issue templates

### PR DESCRIPTION
**Pull request by robbevp**
Adds a basic issue template. This is strongly based on github's default - I guess we'll update and improve as we go along?

re #303

### CODE REVIEW

**Comment by robbevp** [comment]
We might want to create a separate one for questions? I don't think we can expect everyone to jump into the matrix channel. Or can we add the discussion tab like [vuetify](https://github.com/vuetifyjs/vuetify/discussions)?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug_triage
```
Comment (charvp) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Just `bug`, I think.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature_request
```
Comment (charvp) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: We use `enhancement` in the accentor repos.

**Review by charvp — CHANGES_REQUESTED** [review]

**Comment by charvp** [comment]
> We might want to create a separate one for questions? I don't think we can expect everyone to jump into the matrix channel. Or can we add the discussion tab like [vuetify](https://github.com/vuetifyjs/vuetify/discussions)?

I don't like handling questions in GitHub issues. The matrix channel should be accessible to guests, so I don't think people even need to create an account. The discussions tab is a GitHub beta feature, so this can't be enabled (yet). Once it goes into public beta I don't have any qualms about adding it.

**Review by charvp — APPROVED** [review]

**Pull request merged**